src/Microsoft.AspNetCore.Razor.Design/Microsoft.AspNetCore.Razor.Design.csproj

```diff
@@ -57,24 +57,34 @@
     -->
     <MSBuild Projects="@(TaskProject)" />
 
-    <MSBuild Projects="@(TaskProject)" Properties="TargetFramework=net46" Targets="GetTargetPath">
+    <MSBuild Projects="@(TaskProject)" Properties="TargetFramework=net46" Targets="BuiltProjectOutputGroup">
       <Output TaskParameter="TargetOutputs" ItemName="TaskAssemblyNet46" />
     </MSBuild>
 
-    <MSBuild Projects="@(TaskProject)" Properties="TargetFramework=netstandard2.0" Targets="GetTargetPath">
+    <MSBuild Projects="@(TaskProject)" Properties="TargetFramework=net46" Targets="DebugSymbolsProjectOutputGroup">
+      <Output TaskParameter="TargetOutputs" ItemName="TaskSymbolNet46" />
+    </MSBuild>
+
+    <MSBuild Projects="@(TaskProject)" Properties="TargetFramework=netstandard2.0" Targets="BuiltProjectOutputGroup">
       <Output TaskParameter="TargetOutputs" ItemName="TaskAssemblyNetStandard" />
     </MSBuild>
 
-    <Copy SourceFiles="@(TaskAssemblyNet46)" DestinationFolder="$(OutputPath)\tasks\net46\">
+    <MSBuild Projects="@(TaskProject)" Properties="TargetFramework=netstandard2.0" Targets="DebugSymbolsProjectOutputGroup">
+      <Output TaskParameter="TargetOutputs" ItemName="TaskSymbolNetStandard" />
+    </MSBuild>
+
+    <Copy SourceFiles="@(TaskAssemblyNet46);@(TaskSymbolNet46)" DestinationFolder="$(OutputPath)\tasks\net46\">
       <Output TaskParameter="CopiedFiles" ItemName="FileWrites" />
     </Copy>
-    <Copy SourceFiles="@(TaskAssemblyNetStandard)" DestinationFolder="$(OutputPath)\tasks\netstandard2.0\">
+    <Copy SourceFiles="@(TaskAssemblyNetStandard);@(TaskSymbolNetStandard)" DestinationFolder="$(OutputPath)\tasks\netstandard2.0\">
       <Output TaskParameter="CopiedFiles" ItemName="FileWrites" />
     </Copy>
 
     <ItemGroup>
       <None Include="@(TaskAssemblyNet46)" PackagePath="tasks\net46\" Pack="true" />
+      <None Include="@(TaskSymbolNet46)" PackagePath="tasks\net46\" Pack="true" />
```

Comment:
@rynowak we couldn't find any obvious way to produce a symbol nupkg, specify a package path to an item while excluding it only in the main assembly. The result is that we have pdbs sitting in the Design package and it's symbol nupkg. Seems like a small price to pay for compliance. We could open a separate issue to address this in rc if you'd like.

Reply:
Sorry what's the implication of this? I'm not in charge of decisions like this.

Reply:
The implication is after this change we'll have pdbs sitting in both `Microsoft.AspNetCore.Razor.Design.nupkg` and `Microsoft.AspNetCore.Razor.Design.Symbols.nupkg`.